packages/node_modules/pouchdb-adapter-leveldb-core/src/index.js

```diff
@@ -383,7 +383,17 @@ function LevelPouch(opts, callback) {
           return callback(createError(MISSING_DOC, "deleted"));
         }
       } else {
-        rev = opts.latest ? getLatest(opts.rev, metadata) : opts.rev;
+        if (opts.latest) {
+          // #7756: If the revision is a valid pattern but it does not exist, this will throw. In this case just
+          // continue using the non-existing revision, it will not be found and handled as any other (MISSING_DOC).
+          try {
+            rev = getLatest(opts.rev, metadata);
+          } catch (ex) {
+            rev = opts.rev;
+          }
+        } else {
+          rev = opts.rev;
```

Comment:
This seems like it could be 

    rev = opts.rev;
    if (opts.latest) { 
      try { ....

And be a little cleaner?

Reply:
sure, made this change